.azure-pipelines/workbooksValidations.yaml

```diff
@@ -0,0 +1,12 @@
+jobs:
```

Comment:
Please add reference to this validation in "https://github.com/Azure/Azure-Sentinel/blob/master/azure-pipelines.yml" file (the entry point to validation), otherwise this validation won't run.

After adding this change, you will see the new vallidation in "Check" tabs

Reply:
Done. Thanks for the catch!